corehq/apps/data_interfaces/views.py

```diff
@@ -117,6 +117,7 @@ def section_url(self):
         return reverse("data_interfaces_default", args=[self.domain])
 
 
+@location_safe
```

Comment:
I don't think the report actually ensures that the cases being accessed are accessible to restricted users, does it?

Reply:
i think it does. I followed all the additional filters that the current CLE implements...
https://github.com/dimagi/commcare-hq/blob/5a7bec2e848288d22e87a5aa027c5a71070bc1e8/corehq/apps/reports/v2/reports/explore_case_data.py#L145-L147

also the way it fetches `case_owner` is exactly the same as the current CLE.

Reply:
ah okay, perfect - I think I was looking in the wrong place.